lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3OutputStream.java

```diff
@@ -86,6 +88,21 @@ public S3OutputStream(AggregatedMemoryContext memoryContext, S3Client client, S3
         this.requestPayer = context.requestPayer();
         this.sseType = context.sseType();
         this.sseKmsKeyId = context.sseKmsKeyId();
+        this.cannedAcl = getCannedAcl(context.cannedAcl());
+    }
+
+    private ObjectCannedACL getCannedAcl(S3FileSystemConfig.ObjectCannedAcl cannedAcl)
```

Comment:
Let's move this method to the enum. That makes it easier to see how our enum translates to the AWS enum.